CHANGELOG.md

```diff
@@ -9,6 +9,25 @@
 - Test Ref resources to IAM Roles have good assume role documents.  Example: Lambda Function Ref refers to an IAM Role that can be assume by Lambda.
 - More Warnings around hard coded values (Regions, AccountIds) to help with the practice of reusability
 
+### 0.3.0
+###### Features
+- Convert Serverless Transform to SAM
```

Comment:
Does this do this change justice?  Perhaps "Serverless Transforms now handled by SAM libraries"